tests/main/snapctl-is-connected-list/task.yaml

```diff
@@ -0,0 +1,16 @@
+summary: Ensure that snapctl is-connected --list command works.
+
+prepare: |
+  snap pack test-snap
+  snap install --dangerous test-snap_1_all.snap
+
+restore: |
+  snap remove --purge test-snap
+
+execute: |
+  OUT=$(test-snap.listconn 2>&1 || true)
+  EXPECTED=$(printf "network\nx11\n")
+  if [ "$OUT" != "$EXPECTED" ]; then
+    echo "List connections doesn't show expected plugs/slots: $OUT"
+    exit 1
+  fi
```

Comment:
This test is failing on the Ubuntu Core systems, as there is no implicit `x11` slot to connect to, so that's never going to be included in the output. If you want an interface that does exist on Core with the same connection policy, maybe use `network-status`?

It would probably also be useful to have the test continue and disconnect one of the interfaces and demonstrate that the output changes.